CHANGELOG.md

```diff
@@ -11,10 +11,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ### Changed
+
+#### DIRECTORY
+
+- PreconditionFailed error response contains proper ETag and Location headers (see: https://github.com/ehrbase/ehrbase/pull/183)
+
+#### Robot Tests
+
 - Update of AQL-Query test suite (see: https://github.com/ehrbase/ehrbase/pull/179)
 
 ### Fixed
 
+#### DIRECTORY
+
+- Directory IDs from input path or If-Match header must now be in version_uid format (see https://github.com/ehrbase/ehrbase/pull/183)
+- Folder IDs inside body are now parsed correctly (see: https://github.com/ehrbase/ehrbase/pull/183)
+
+#### Robot Tests
```

Comment:
Quick comment: I'm not a fan of those sub-headers. Would you argue we benefit from them? I think this is just one layer more of where stuff will be added wrong into the changelog ;) For instance, the linked #207 below isn't just a robot test update, instead it introduced the validation of EHR_STATUS object and fixed other bugs.